sql-plugin/src/main/scala/org/apache/spark/sql/rapids/datetimeExpressions.scala

```diff
@@ -1044,29 +1044,34 @@ class FromUTCTimestampExprMeta(
     rule: DataFromReplacementRule)
   extends BinaryExprMeta[FromUTCTimestamp](expr, conf, parent, rule) {
 
+  private[this] var timezoneId: ZoneId = null
+
+  lazy val supportedZoneStrList = Seq("UTC", "Asia/Shanghai")
```

Comment:
Could we move this to `TimeZoneDB`.
'+08:00' is also supported.
We can add a method to check if we support:
```
def supportTZ(tz: String) =  {
  val rules =  ZoneId.of(tz, ZoneId.SHORT_IDS).getRules
  // '+08:00' isFixedOffset
  (rules.isFixedOffset || rules.getTransitionRules.isEmpty)
}

```